reference-architecture/vmware-ansible/roles/docker-storage-setup/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: create the docker-storage-setup config file
+  copy:
+     src: docker-storage-setup
+     dest: /etc/sysconfig/docker-storage-setup
```

Comment:
I thought about it, but I think if you were to customize the size of your attached VMDKs you would be customizing the size overall. The VMware template in question is used by all of the VMs. The default option is thin provisioned and 40GB. But, you could size it however you like. Did you mean to customize the drive? sdc vs. sdb or whatever?